### ISSUE TITLE
Code no longer works on chrome

### ISSUE THREAD

**Issue by g-casey** [TACIXAT/XorShift128Plus#5]
No error messages besides unsat printing to the console.  Happens with any numbers I use on chrome.  Works on firefox every time though.

**Comment by gavemyheart** [comment]
I can confirm this issue.

**Comment by TACIXAT** [comment]
Happy to merge any pull requests to fix this.

**Comment by TACIXAT** [comment]
I'm guessing this commit is the one to match.

https://github.com/v8/v8/commit/ac66c97cfddc1e9fd89b494950ecf8a1a260bc80#diff-202872834c682708e9294600f73e4d15

**Comment by gerald-dotcom** [comment]
@TACIXAT I wish I could but I lack the expertise in C++

Happy to donate tho 

This project is great.

**Comment by TACIXAT** [comment]
Should work now. Chrome switched from an XorShift128Plus to a simple XorShift128 :)

https://github.com/v8/v8/commit/ac66c97cfddc1e9fd89b494950ecf8a1a260bc80#diff-202872834c682708e9294600f73e4d15R111

Basically it was `state0 + state1`, and got changed to `state0 >> 12` as the main portion of the random number output.  Hopefully I didn't break Firefox, but yea, I think I got all the spots right. Worked for me on 5 doubles, I guess the new method has a fewer bits.
